bbb-graphql-actions/src/actions/chatDeleteMessageReaction.ts

```
@@ -0,0 +1,32 @@
+export default function buildRedisMessage(sessionVariables: Record<string, unknown>, input: Record<string, unknown>): RedisMessage {
+  throwErrorIfInvalidInput(input,
+      [
+        {name: 'chatId', type: 'string', required: true},
+        {name: 'messageId', type: 'string', required: true},
+        {name: 'reactionEmoji', type: 'string', required: true},
+      ]
+  )
+
+  const eventName = `DeleteGroupChatMessageReactionReqMsg`;
+  const routing = {
+    meetingId: sessionVariables['x-hasura-meetingid'] as String,
```

Biome

[error] Don't use 'String' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'string' instead


(lint/complexity/noBannedTypes)